client/api_handler.py

```diff
@@ -58,6 +62,13 @@ def get_metadata(url, response_type=ResponseType.JSON):
             return get_json_metadata(metadata.data_map)
 
     metadata.parse_content(site_response)
+
+    # Trim the description if necessary
+    if metadata.prop_map['description'] is not None:
```

Comment:
I think we should put this in metadata? or at least extract out into it's own function